File: src/main/java/dev/salmon/seraph/listener/LocrawListener.java
package dev.salmon.seraph.listener;

import dev.salmon.seraph.listener.event.LocrawEvent;
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;

public class LocrawListener {
    @SubscribeEvent
    public void onJoin(LocrawEvent.JoinGame event) {
        System.out.println("joined gametype: " + event.locraw.getGameType().toString() + " mode: " + event.locraw.getGameMode());
    }

    @SubscribeEvent
    public void onJoinLobby(LocrawEvent.JoinLobby event) {
        System.out.println("joined lobby " + event.locraw.getGameType().toString());
    }
}

File: src/main/kotlin/com/github/scherso/seraph/listener/LocrawListener.kt
package com.github.scherso.seraph.listener

import club.maxstats.weave.loader.api.event.SubscribeEvent
import com.github.scherso.seraph.event.LocrawEvent

class LocrawListener {

    @SubscribeEvent
    fun onJoinGame(event: LocrawEvent.JoinGame) {
        System.out.printf("%s-5s%s-5s%s", "GAME", event.locraw.gameType, "MODE", event.locraw.gameMode)
    }

    @SubscribeEvent
    fun onJoinLobby(event: LocrawEvent.JoinLobby) {
//      Minecraft.getMinecraft().thePlayer.addChatMessage(ChatComponentText("You have joined the ${event.locraw.gameType} lobby."))
        System.out.printf("%s-5s%s-5s%s", "LOBBY", event.locraw.gameType, "MODE", event.locraw.gameMode)
    }
}
